authapi/permissions.py

```diff
@@ -19,7 +20,9 @@ def __init__(self, permission_type):
     def has_permission(self, permission, request, view):
         user = request.user
         permissions = get_user_permissions(user)
-        permissions = find_permission(permissions, self.permission_type)
+        permissions = find_permission(
+            permissions, self.permission_type,
+            namespace=settings.PERMISSION_NAMESPACE)
```

Comment:
Yes, for at least one of the views (probably multiple), we have tests that test generic `foo` namespaces as well as the `__auth__` namespaces.